src/sentry/apidocs/parameters.py

```diff
@@ -158,6 +160,69 @@ class OrganizationParams:
     )
 
 
+class ReleaseParams:
+    VERSION = OpenApiParameter(
+        name="version",
+        location="path",
+        required=True,
+        type=str,
+        description="The version identifier of the release",
+    )
+    PROJECT_ID = OpenApiParameter(
+        name="project_id",
+        location="query",
+        required=False,
+        type=str,
+        description="The project id to filter by.",
+    )
+    HEALTH = OpenApiParameter(
+        name="health",
+        location="query",
+        required=False,
+        type=bool,
+        description="Include health data with the release.",
```

Comment:
```suggestion
        description="Whether or not to include health data with the release. By default, this is ___",
```
Since it's not required, what's the default behavior here?